microsoft-edge/devtools-guide-chromium/performance/reference.md

```diff
@@ -0,0 +1,1238 @@
+
+![The 'Save profile' right-click menuitem](./reference-images/save-profile.png)
+
+
+<!-- ------------------------------ -->
+#### Load a recording
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#load -->
+
+To load a recording from a file, click the **Load profile** (![The Load profile icon](./reference-images/load-profile-icon.png)) button in the action bar at the top of the **Performance** tool.
+
+![Load trace button via button in the action bar](./reference-images/load-profile-via-button.png)
+
+The **Performance** tool will show annotations if they are present in the trace.
+
+Or, right-click in the **Performance** tool, and then select **Load profile**:
+
+![The 'Load profile' right-click menuitem](./reference-images/load-profile-via-right-click.png)
+
+You can right-click in an existing recording, or on the screen that's displayed in the **Performance** tool when there are no recordings.
+
+
+<!-- ====================================================================== -->
+## Clear the previous recording
+
+After making a recording, to remove that recording from the **Performance** tool, click the **Clear recording** (![Clear recording icon](./reference-images/clear-recording-icon.png)) button:
+
+![Clear recording](./reference-images/clear-button.png)
+
+
+<!-- ====================================================================== -->
+## Analyze a performance recording
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#analyze -->
+
+After you [record runtime performance](#record-runtime-performance) or [record load performance](#record-load-performance), the **Performance** tool displays a lot of data about the recording.  Use the recorded data to analyze the performance of your webpage.
+
+
+<!-- ------------------------------ -->
+#### Get actionable insights
+
+The **Performance** tool extracts actionable insights from the recorded performance data.  These insights are ways in which you can improve the performance of your website.  The insights are focused on:
+
+* LCP and INP by subpart.
+* LCP request discovery.
+* Layout shift culprits.
+* Render blocking requests.
+* Third parties.
+* Image delivery.
+* Document request latency.
+* Viewport optimization for mobile.
+* CSS selector costs.
+
+To use the insights:
+
+1. Make a performance recording, per [Record performance](#record-performance), above.
+
+1. In the left sidebar of the **Performance** tool, select the **Insights** tab, expand different sections, and hover over and click items.  The **Performance** tool highlights the corresponding events in the trace:
+
+   ![Hovering over entries in the Insights tab ](./reference-images/insights-tab.png)
+
+
+   To make the **Interactions** tab visible, click the **Move activity bar to the left** (![The "Move activity bar to the left" icon](./reference-images/move-activity-bar-to-the-left-icon.png)) button.<!-- added in Edge doc -->
```

Comment:
I don't understand this line. This section does not talk about the Interactions tab. Furthermore, why would changing the Activity Bar location have an impact on the Interactions tab?

Reply:
changed to what was probably the intention here:
```
If the sidebar and **Insights** tab is not visible, click the **Show sidebar** (![The Show sidebar icon](./reference-images/show-sidebar-icon.png)) button.
```
@captainbrosset 

like in https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/perf-sync/microsoft-edge/devtools-guide-chromium/performance/reference.md#annotate-a-recording-and-share-it